lib/Horizon.dart

```diff
@@ -0,0 +1,224 @@
+                                                            Colors.grey[800]!,
+                                                            Colors.transparent,
+                                                        ],
+                                                    ),
+                                                ),
+                                            ),
+                                            Image.network(
+                                                dailyForecast.imageId,
+                                                fit: BoxFit.cover,
+                                            ),
+                                            Center(
+                                                child: Text(
+                                                    dailyForecast.getDate(currentDate.day).toString(),
+                                                    style: textTheme.headline4,
+                                                ),
+                                            ),
+                                        ],
+                                    ),
+                                ),
+                                Expanded(
+                                    child: Padding(
+                                        padding: EdgeInsets.all(6.0),
+                                        child: Column(
+                                            crossAxisAlignment: CrossAxisAlignment.start,
+                                            children: <Widget>[
+                                                Text(
+                                                    dailyForecast.getWeekday(currentDate.weekday),
+                                                    style: textTheme.headline5,
+                                                ),
+                                                SizedBox(height: 10.0),
+                                                Text(dailyForecast.description),
+                                            ],
+                                        ),
+                                    )
+                                ),
+                                Padding(
+                                    padding: EdgeInsets.all(6.0),
+                                    child: Text(
+                                        "${dailyForecast.highTemp} | ${dailyForecast.lowTemp} F",
+                                        style: textTheme.subtitle2,
+                                    ),
+                                ),
+                            ],        
+                        ),
+                    );
+                },
+                childCount: 7,
+            ),
+        ); 
+    }
+}
+
+const String baseAssetURL =
+    'https://dartpad-workshops-io2021.web.app/getting_started_with_slivers/';
+const String headerImage = '${baseAssetURL}assets/header.jpeg';
+
+const Map<int, DailyForecast> _kDummyData = {
```

Comment:
Comment: This variable name sounds odd. Why not just `dummyData` or something more descriptive `weatherData` ?